site/docs/v1/tech/apis/_reactor-response-body.adoc

```diff
@@ -0,0 +1,69 @@
+==== Response Body
+
+[.api]
+[field]#metrics.breachedPasswordMetrics# [type]#[Map<UUID, List<Object>]#::
+The breached password metrics. The map key is the Tenant Id.
+
+[field]#metrics.breachedPasswordMetrics``[tenantId]``.actionRequired# [type]#[Integer]#::
+The total number of user accounts where action is required.
+
+[field]#metrics.breachedPasswordMetrics``[tenantId]``.matchedCommonPasswordCount# [type]#[Integer]#::
+The total number of matched user accounts with a commonly compromised password.
+
+[field]#metrics.breachedPasswordMetrics``[tenantId]``.matchedExactCount# [type]#[Integer]#::
+The total number of matched user accounts with a compromised password exact match.
+
+[field]#metrics.breachedPasswordMetrics``[tenantId]``.matchedPasswordCount# [type]#[Integer]#::
+The total number of user accounts with matched passwords.
+
+[field]#metrics.breachedPasswordMetrics``[tenantId]``.matchedSubAddressCount# [type]#[Integer]#::
+The total number of sub-email user accounts that match a compromised password.
+
+[field]#metrics.breachedPasswordMetrics``[tenantId]``.passwordsCheckedCount# [type]#[Integer]#::
+The total number of user account passwords checked.
+
+[field]#status.advancedIdentityProviders# [type]#[String]#::
```

Comment:
This is really nice, gives a nice way to know what all the paid edition features are!